src/app/src/main/kotlin/org/imperial/mrc/hint/exceptions/HintExceptionHandler.kt

```diff
@@ -75,12 +75,37 @@ class HintExceptionHandler(private val errorCodeGenerator: ErrorCodeGenerator,
         return translatedError(error.key, error.httpStatus, request)
     }
 
+    @ExceptionHandler(AdrException::class)
+    fun handleAdrException(error: AdrException, request: HttpServletRequest): ResponseEntity<Any>
+    {
+        logger.error(request, error)
+
+        val messageArguments = arrayOf(error.adrUri ?: "")
```

Comment:
Pretty minor but can we make the error text link to the resource instead of just showing the URL?

Like
Unable to load resource, [check resource in ADR](https://adr.unaids.org/country-estimates-23/antarctica-country-estimates-2023-1/resource/ec4f40f7-601e-4f1c-bcb2-4643da4cc4b5).